### DISCUSSION TITLE
Спец. вид по условию

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#226] [Как это сделать?] [answered]
Есть возможность отключать и включать спец. вид при использовании Тип таблицы - По другой таблице в поле Селект дерево?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Не, дерево и так на пределе понимания целевой аудитории. Да и для нас — это тоже катастрофически сложный механизм. Либо есть, либо нет.
